<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SunnyTcsProcedureConfigDig</class>
 <widget class="QWidget" name="SunnyTcsProcedureConfigDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>843</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QLabel" name="proName">
     <property name="text">
      <string>流程名</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>流程名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ProcedureNameEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="proExplain"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="saveProcedureBtn">
         <property name="text">
          <string>保存流程</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="createActionBtn">
         <property name="text">
          <string>添加动作</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="deleteActionBtn">
         <property name="text">
          <string>删除动作</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,2,0">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>目标标签</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>动作类型</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QTableWidget" name="tableWidget">
           <column>
            <property name="text">
             <string>目标标签</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>文件路径</string>
            </property>
           </column>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>执行次数</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>动作说明</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>动作名</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="exeCountSBox"/>
         </item>
         <item row="0" column="2">
          <widget class="QComboBox" name="Action_CBox">
           <item>
            <property name="text">
             <string>修改值</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>添加xml标签</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>删除xml标签</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>属性修改</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>添加json对象/值</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>删除json对象/值</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="ActionNameEdit"/>
         </item>
         <item row="4" column="2">
          <widget class="QPlainTextEdit" name="ActionExplainPlainEdit"/>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="deleteSelectedItemBtn">
           <property name="text">
            <string>删除选中标签</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>数据类型</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QComboBox" name="A0_valueTypeCBox">
           <item>
            <property name="text">
             <string>文本</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>整型数</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="extraPageSWidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page">
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>161</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>添加xml附页</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="xmlAttEdit">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>100</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="attAddBtn">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>130</y>
             <width>61</width>
             <height>28</height>
            </rect>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="xmlNameEdit">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>70</y>
             <width>121</width>
             <height>21</height>
            </rect>
           </property>
          </widget>
          <widget class="QListWidget" name="xmlattsList">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>170</y>
             <width>201</width>
             <height>211</height>
            </rect>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>70</y>
             <width>72</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>标签名</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>100</y>
             <width>72</width>
             <height>15</height>
            </rect>
           </property>
           <property name="text">
            <string>标签属性</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>添加Json附页</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="addJsonCBox">
             <item>
              <property name="text">
               <string>对象</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>值</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QStackedWidget" name="addJsonSWidget">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page_3">
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLabel" name="label_11">
                 <property name="text">
                  <string>值列表</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QLineEdit" name="jsonValueEdit"/>
                   </item>
                   <item>
                    <widget class="QComboBox" name="jsonValueCBox">
                     <item>
                      <property name="text">
                       <string>文本</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>数值</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QPushButton" name="jsonValueBtn">
                   <property name="text">
                    <string>添加</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QTableWidget" name="addJsonTable">
                 <column>
                  <property name="text">
                   <string>值名</string>
                  </property>
                 </column>
                 <column>
                  <property name="text">
                   <string>类型</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_4"/>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_5"/>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addActionBtn">
         <property name="text">
          <string>保存动作</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="openFileBtn">
           <property name="text">
            <string>打开文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QMdiArea" name="mdiArea"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
